test/load/fleet-autoscaling/locust-files/locustfile.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from locust import HttpLocust, TaskSet, events, task
+import locust.events
+import json
+import time
+import socket
+import atexit
+
+FLEET_SIZE = 100
+DEADLINE = 30 * 60
+
+class UserBehavior(TaskSet):  
+    @task
+    def scaleUpFleet(self):
+		# Create a fleet.
+		initial_size = 1
+		start_time = time.time()
```

Comment:
nit: format 'payload' as multiline ?

Reply:
Formatted as multiline for now. I will later move these to JSON files but not in this PR since it needs some parsing to build the payload.